lib/components/src/typography/DocumentFormatting.tsx

```diff
@@ -71,10 +72,16 @@ export const Pre = styled.pre<{}>(withReset, withMargin, ({ theme }) => ({
 }));
 
 const Link: FunctionComponent<any> = ({ href: input, children, ...props }) => {
+  // If storybook is hosted at a non-root path (e.g. `/storybook/`),
+  // the base url needs to be prefixed to storybook paths.
+  let storybookBaseUrl =
+    typeof window !== 'undefined' ? window.parent.document.location.pathname : '/';
```

Comment:
This access should be safe, even if the content is rendered outside of an iframe.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/parent):

> If a window does not have a parent, its parent property is a reference to itself.